reference/content/driver-scala/tutorials/aggregation.md

```diff
@@ -94,6 +96,7 @@ To [explain]({{< docsref "reference/command/explain/" >}}) an aggregation pipeli
 method:
 
 ```java
+// Note: this code example uses a custom implicit helper referenced in the Quick Start Primer
```

Comment:
@rozza 
This code example looks like it's using Java instead of Scala. I noticed that the code included Arrays.asList() instead of Seq() and the "java" syntax highlighting in markdown.

Could you provide me with Scala code to fix this example? Thanks!

Reply:
```scala
// Note: this code example uses a custom implicit helper referenced in the Quick Start 
collection.aggregate(
  Seq(Aggregates.filter(Filters.eq("categories", "Bakery")), 
      Aggregates.group("$stars", Accumulators.sum("count", 1))))
  .explain()
  .printResults()
```

Good catch - as match is a reserved word in scala, there is a `filter` overload for scala users.